### ISSUE TITLE
Program test cases do not pass

### ISSUE THREAD

**Issue by liqiang4113** [HawkAaron/warp-transducer#20]
Thank you for your codes.
I have a problem GPU version warp-transducer.
Source code compilation is ok, but the program test (build/test_gpu) did not respond.

thank you

pytorch 1.0.1
centos 7
cuda 10
Tesla V100-PCIE

**Comment by HawkAaron** [comment]
@liqiang4113 CUDA10 may not support yet, please try CUDA9.0

**Comment by tksj09** [comment]
@HawkAaron  
I have a same problem.Tesla V100 , and cuda 9.0 ,the program test (build/test_gpu) also did not respond

**Comment by liqiang4113** [comment]
V100 :The program does not respond to test. How to adjust in codes?  
CUDA10 : loss value is sometimes nan but the learning rate is already very low

**Comment by liqiang4113** [comment]
@HawkAaron Thanks for your help. Tesla V100 ,I wonder  why the test program does not pass on such a high configuration device. What is the problem and can you give us some pointers?

**Comment by HawkAaron** [comment]
@liqiang4113 Could you use warp-ctc in Tesla V100 ? Since don't have this kind of device, I may not be able to help.

**Comment by liqiang4113** [comment]
@HawkAaron Thanks. Warp-ctc is OK in Tesla V100. When running the following line of code, there is no response and No error messages are generated. costs = rnnt_loss(logits, labels, input_length, label_length, blank) (test_basic.py)

**Comment by HawkAaron** [comment]
There were some updates recently, could you please run the test again ?
